modules/sections.js

```diff
@@ -0,0 +1,15 @@
+export function changeSection() {
```

Comment:
- [ ] **Required:** Kindly change this function as well to use `ES6 arrow function`.

Reply:
Fixed this as well, thank you!